code/modules/mob/living/basic/space_fauna/demon/demon_subtypes.dm

```diff
@@ -0,0 +1,163 @@
+/// The famous blood-crawling slaughter demons of wizardry fame.
+/mob/living/basic/demon/slaughter
+	name = "slaughter demon"
+	real_name = "slaughter demon"
+	unique_name = FALSE
+	speak_emote = list("gurgles","wails","screeches")
+
+	icon_state = "slaughter_demon"
+	icon_living = "slaughter_demon"
+
+	minimum_survivable_temperature = TCMB
+	obj_damage = 50
+	melee_damage_lower = 15 // reduced from 30 to 15 with wounds since they get big buffs to slicing wounds
+	melee_damage_upper = 15
+	wound_bonus = -10
+	bare_wound_bonus = 0
+	sharpness = SHARP_EDGED
```

Comment:
Is `/mob/living/basic/demon` actually used anywhere? Why the differentiation between these stats between the two types, if no?
Also we can lose the historical comment on `melee_damage_lower`.

Reply:
i feel like it was around for historical reasons and then the slaughter demon was built on top of it. i still think there's value to admins/etc. to have a demon-esque creature that can't automatically bloodcrawl/have the killstreak functionality.